pom.xml

```diff
@@ -34,16 +34,15 @@
         <changelist>-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/clover-plugin</gitHubRepo>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.level>8</java.level>
-        <jenkins.version>2.289.3</jenkins.version>
+        <jenkins.version>2.387.3</jenkins.version>
```

Comment:
Can you add 

```##Unreleased

* Minimum Jenkins version required is 2.387.3 now```

in the CHANGELOG.md?

FYI: I'm about to release new Jenkins Clover Plugin version, 4.14.0, very soon.

Reply:
Happy to do that.  Have you considered switching to release drafter so that the changelog is maintained automatically?  [Instructions](https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc) are available with more details.

Added in 7b31e02af059157660798a195721fb571bb236cc

Reply:
I also changed the title of the pull request so that the release drafter will include the "Require Java 11 and Jenkins 2.387.3 or newer" in the changelog that it generates.

Reply:
Thank you, yes I'm working on automation and release drafter, but this is not active yet.